tsconfig.json

```diff
@@ -2,9 +2,6 @@
   "include": [
     "src/**/*.ts"
   ],
-  "exclude": [
```

Comment:
return this to tsconfig

Reply:
This code does nothing as only content of `src` folder included